src/features/COVID/COVID-API.js

```diff
@@ -0,0 +1,16 @@
+// A mock function to mimic making an async request for data
```

Comment:
- [ ] App has unit tests for several components of the app and some pure functions in Redux code. You only made an integration test for your project. Kindly add unite tests as well.